test/onnx/error_reproduction.py

```diff
@@ -0,0 +1,140 @@
+"Error reproduction utilities for op consistency tests."
+
+from __future__ import annotations
+
+import difflib
+import pathlib
+import sys
+import time
+import traceback
+import torch
+
+_MISMATCH_MARKDOWN_TEMPLATE = """\
+### Summary
+
+The output of ONNX Runtime does not match that of PyTorch when executing test
+`{test_name}`, `sample {sample_num}` in ONNX Script `TorchLib`.
+
+To recreate this report, use
+
+```bash
+CREATE_REPRODUCTION_REPORT=1 python -m pytest onnxscript/tests/function_libs/torch_lib/ops_test.py -k {short_test_name}
+```
+
+### Inputs
+
+Shapes: `{input_shapes}`
+
+<details><summary>Details</summary>
+<p>
+
+```python
+kwargs = {kwargs}
+inputs = {inputs}
+```
+
+</p>
+</details>
+
+### Expected output
+
+Shape: `{expected_shape}`
+
+<details><summary>Details</summary>
+<p>
+
+```python
+expected = {expected}
+```
+
+</p>
+</details>
+
+### Actual output
+
+Shape: `{actual_shape}`
+
+<details><summary>Details</summary>
+<p>
+
+```python
+actual = {actual}
+```
+
+</p>
+</details>
+
+### Difference
+
+<details><summary>Details</summary>
+<p>
+
+```diff
+{diff}
+```
+
+</p>
+</details>
+
+### Full error stack
+
+```
+{error_stack}
+```
+"""
```

Comment:
I would also include:

1. The exported fx graph
2. The ONNX model

Reply:
Hmm adding FX graph to ONNXProgram is a big change

Reply:
Added onnx model